packages/govtnz-ds-website/src/pages/prototypes/index.tsx

```diff
@@ -224,6 +243,7 @@ class ContactusForm extends React.Component {
 
           <TextareaBlock
             autoComplete="example"
```

Comment:
The autoComplete attribute is per the HTML5 spec https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete